packages/svelte/preprocess.js

```diff
@@ -6,7 +7,7 @@ const {parse} = require('@babel/parser');
 const {KEYS} = require('@reshadow/core');
 const reshadow = require('@reshadow/babel');
 
-const {stripIndent} = require('common-tags');
+const stripIndent = require('strip-indent');
```

Comment:
I was using `common-tags`, because it has some other useful helpers. `common-tags` is also used by other packages in reshadow (babel, postcss, prettier), and maybe it would be better to replace this package for them too?

but is `strip-indent` really better? 🙂

Reply:
It's faster (;